.gitmodules

```diff
@@ -15,7 +15,7 @@
 	url = https://github.com/purpleidea/go-systemd
 [submodule "vendor/honnef.co/go/augeas"]
 	path = vendor/honnef.co/go/augeas
-	url = https://github.com/dominikh/go-augeas/
```

Comment:
Why are we removing the trailing slashes?

Reply:
Some git compat issue, but this is no longer part of this patchset anyway.